docs/changelog/117154.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 117154
+summary: "Add CCS based reindex to docs"
+area: Search
+type: bug
```

Comment:
This is not required since we're just adding documentation, and the original [PR](https://github.com/elastic/elasticsearch/pull/96968) had a changelog. Please remove this file from the PR.

Reply:
[omit changelog](https://github.com/elastic/elasticsearch/pull/117154/commits/69985f7d920aa1eb2ea3f0e8e48f17269364f655)